doc/spinel-protocol-src/spinel-prop-net.md

```diff
@@ -55,8 +55,16 @@ Values:
 
 The partition ID of the partition that this node is a member of.
 
-### PROP 73: PROP_NET_KEY_SWITCH_GUARDTIME   {#prop-net-key-swtich-guardtime}
+### PROP 73: PROP_NET_REQUIRE_JOIN_EXISTING   {#prop-net-require-join-existing}
```

Comment:
It appears PROP 73 was originally `PROP_NET_REQUIRE_JOIN_EXISTING`, but was lost in the documentation with #534.